tests/bin/no_doc/run.t

```diff
@@ -3,7 +3,8 @@ We need a basic opam project skeleton
     $ cat > CHANGES.md << EOF \
     > ## 0.1.0\
     > \
-    > - Some other feature\
+    >  - Change A\
+    >  - Change B\
```

Comment:
Maybe we should add trailing whitespaces here to make sure they're being dropped!

Although I guess this is also the kind of things we could extract a function for and have finer grained unit tests. What do you think?

Reply:
Good idea, done.